src/data_processing/message_handler.rs

```diff
@@ -90,15 +115,15 @@ pub async fn clean_up_dedupe_queue(
 pub async fn watch_received_message_queue(
     mut input_queue: Receiver<serde_json::Value>,
     output_queue: Sender<serde_json::Value>,
-    config: &MessageHandlerConfig,
+    config: MessageHandlerConfig,
```

Comment:
Changed this from taking a pointer to consuming the object as the object being passed in wasn't being used elsewhere.